common/ideas/00_basic_ideas.txt

```diff
@@ -705,350 +673,369 @@ maritime_ideas = {
+	trigger = {
+		OR = {
+			has_government_attribute = enables_aristocratic_idea_group
+			government = theocracy
+			government = tribal
+		}
 	}
-	factories = {
-		rival_border_fort_maintenance = -0.5
-		center_of_trade_upgrade_cost = -0.2
+
+	noble_knights = {
+		cavalry_power = 0.10
+		leader_land_shock = 1
 	}
-	additional_diplomats = {
-		diplomatic_upkeep = 1
+	military_traditions = {
+		mil_tech_cost_modifier  = -0.2
+		army_tradition_decay = -0.01
 	}
-	general_colonization_law  = {
-		colonists = 1
-		colonist_placement_chance = 0.05
+	local_nobility = {
+		global_autonomy = -0.025
+		yearly_absolutism = 0.5
 	}
-	competetive_merchants  = {
-		global_trade_power = 0.2
+	serfdom = {
+		global_manpower_modifier = 0.33
+	}
+	noble_officers = {
+		free_leader_pool = 2
+		general_cost = -0.66
+	}
+	international_nobility = {
+		fire_damage_received = -0.15
+	}
+	noble_resilience = {
+		mercenary_manpower = 0.20
 	}
 
+
 	ai_will_do = {
-		factor = 0.48
-		modifier = {
-			factor = 10
-			OR = {
-				has_idea_group = exploration_ideas
-				is_colonial_nation = yes
-			}
-		}
-		modifier = {
-			factor = 5
-			OR = {
-				tag = CAS
-				tag = POR
-				tag = SPA
-			}
-		}
-		modifier = {
-			factor = 10
-			NOT = { has_dlc = "Third Rome" } # Has siberian frontier instead
-			tag = RUS
-			num_of_cities = 20
-			any_owned_province = {
-				has_empty_adjacent_province = yes
-			}
-		}
-		modifier = {
-			factor = 0
-			NOT = { has_idea_group = exploration_ideas }
-			NOT = { is_colonial_nation = yes }
-			NOT = {
-				any_owned_province = {
-					has_empty_adjacent_province = yes
-				}
-			}
-		}
+		factor = 0.45
+
 		modifier = {
-			factor = 0.1
-			NOT = {
-				technology_group = western
-				technology_group = eastern
-				technology_group = muslim
-			}
+			factor = 0.5
+			is_subject = yes
 		}
 	}
 }
 
 
-administrative_ideas = {
-	category = ADM
+plutocracy_ideas = {
+	category = MIL
 
 	bonus = {
-		governing_capacity_modifier = 0.25
+		global_institution_spread = 0.10
+		burghers_loyalty_modifier = 0.1
+		vaisyas_loyalty_modifier = 0.1
 	}
 
-	organised_mercenary_payment = {
-		mercenary_cost = -0.10
+	trigger = {
+		has_government_attribute = enables_plutocratic_idea_group
 	}
-	adaptability = {
-		core_creation = -0.25
+
+	tradition_of_payment = {
+		mercenary_manpower = 0.10
+		mercenary_discipline = 0.025
 	}
-	benefits_for_mercenaries = {
-		merc_maintenance_modifier = -0.15
+	abolished_serfdom = {
+		land_morale = 0.10
 	}
-	bookkeeping = {
-		interest = -0.5
+	local_militia = {
+		global_unrest = -1
+		global_regiment_recruit_speed = -0.33
 	}
-	organised_mercenary_recruitment = {
-		mercenary_manpower = 0.25
+	conscription = {
+		global_manpower 0.2
+		movement_speed = 0.15
 	}
-	administrative_efficiency_idea = {
-		advisor_pool = 1
+	guerrilla_tactics = {
+		movement_speed = 0.15
+		shock_damage = 0.10
 	}
-	civil_servants = {
-		adm_tech_cost_modifier = -0.1
+	active_reserves = {
+		reserves_organisation = 0.40
```

Comment:
Very, very strong I think. I would consider toning this down a little bit, and giving instead bumping merc discipline to 5%